### PR TITLE
Manually fix files that black could not automatically reformat

### PR DESCRIPTION
**Pull request by clacroix12**
Signed-off-by: Coady LaCroix <clacroix@redhat.com>

### CODE REVIEW

**Comment by clacroix12** [comment]
This PR is to set our repository up so we can use black to format everything in one go. This commit was cherry picked from #3234 which is no longer going to be merged and instead split into a few different PRs, this being the first.

**Review by vasukulkarni — APPROVED** [review]

**Review by mbukatov — APPROVED** [review]

File: ocs_ci/utility/lvm_utils.py [hunk 1]
```diff
@@ -30,64 +30,82 @@ def osd_scenario1(osd, devices_dict, dmcrypt=False):
...
+        osd.LvmConfig.vg_name %
+        '1',
+        osd.LvmConfig.size.format(2)
+    )
+
+    scenario = (
+        "{{'data':'{datalv1}','data_vg':'{vg_name}'}},{{'data':'{datalv2}','data_vg':'{vg_name}',"
+        "'db':'{dblv1}','db_vg':'{vg_name}'}},"
+        "{{'data':'{datalv3}','data_vg':'{vg_name}','wal':'{wallv1}','wal_vg':'{vg_name}'}},"
+        "{{'data':'{datalv4}','data_vg':'{vg_name}','db':'{dblv2}','db_vg':'{vg_name}','wal':'{wallv2}',"
```
Comment (petr-balogh) [inline ocs_ci/utility/lvm_utils.py:102]: I see this line is still quite long, can we maybe break this line as well?

**Review by petr-balogh — APPROVED** [review]
Approving and not blocking this one. Maybe this long line will be then fixed by black in next PR you are going to send ;)

File: ocs_ci/utility/lvm_utils.py (hunk 1 above)
Comment (clacroix12) [reply ocs_ci/utility/lvm_utils.py:102]: Black will organize the `.formats()` much better but not the strings if they are too long. I will eventually cover the long strings in subsequent PRs.

**Pull request merged**